Utilities/sandbox/parse-manifest.sb

```diff
@@ -0,0 +1,19 @@
+(version 1)
+
+; Deny everything by default.
+(deny default)
+
+; Allow reading all files.
+(allow file-read*)
```

Comment:
Yeah; we also currently haven't said we don't support importing frameworks from the SDK even though we don't really want anyone to